src/go/util/mm/minimega.go

```diff
@@ -761,6 +762,20 @@ func (Minimega) IsC2ClientActive(opts ...C2Option) error {
 	}
 }
 
+
+func (mm Minimega) IsC2ClientActive(opts ...C2Option) error {
+	o := NewC2Options(opts...)
+	if o.skipActiveClientCheck {
+		return nil
+	}
+
+	vms := GetVMInfo(NS(o.ns), VMName(o.vm))
+	if len(vms) == 0 {
+		return fmt.Errorf("VM %s does not exist", o.vm)
+	}
+	return mm.isC2ClientActive(vms[0], opts...)
```

Comment:
@jacdavi why did you break this into two separate functions?

Reply:
Yeah exactly, not the most elegant though. For putting `cc active` in the table, we'd definitely want the better check in place for a list call to `GetVmInfo` as opposed to just limiting to one vm.